client/dashboard/data/types.ts

```diff
@@ -138,9 +138,30 @@ export interface EngagementStats {
 	previousData: EngagementStatsDataPoint;
 }
 
-export type PerformanceReport = {
+export interface SiteSettings {
+	settings: {
+		wpcom_performance_report_url?: string;
+	};
+}
+
+export interface PerformanceInsightsQueryState {
```

Comment:
While it's not introduced in this particular PR, I think this type and PerformanceReport and UrlPerformanceInsights are "application types", so they don't belong in `data/types`.

We should be thinking about `data` folder as a data layer separate from the application code.

Reply:
Good call out.

I can clean that up now. Where would you expect that type to live? In the queries.ts?

Reply:
I think the types should live close to their usage (if there's just a single usage) in general (so either query or component).
That said, for this one, the type might not be needed at all actually. The right type should be 

```
import type { Query } from '@tanstack/react-query';

Query< UrlPerformanceInsights >
```